core/src/index/knowhere/knowhere/index/vector_index/IndexIVFSQNR.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2019-2020 Zilliz. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License
+// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+// or implied. See the License for the specific language governing permissions and limitations under the License
+
+#pragma once
+
+#include <memory>
+#include <utility>
+
+#include "knowhere/index/vector_index/IndexIVF.h"
+
+namespace milvus {
+namespace knowhere {
+
+class IVFSQNR : public IVF {
+ public:
+    IVFSQNR() : IVF() {
+        index_type_ = IndexEnum::INDEX_FAISS_IVFSQ8;
+    }
```

Comment:
SQ8NR ?

Reply:
In knowhere we use IVFSQ() for ivfsq8 index, similarly it should be IVFSQNR()